source/_integrations/lightwave.markdown

```diff
@@ -37,14 +37,84 @@ lightwave:
       name: Torch socket
 ```
 
+{% configuration %}
+host:
+  description: IP address of your Lightwave hub
+  required: true
+  type: string
+lights:
+  description: List of lights you wish to configure
+  required: false
+  type: map
+  keys:
+    name:
+      description: Name of the Light
+      required: true
+      type: string
+switches:
+  description: List of switches you wish to configure
+  required: false
+  type: map
+  keys:
+    name:
+      description: Name of the Switch
+      required: true
+      type: string
+trv:
+  description: PLEASE ADD A DESCRIPTION HERE :)
```

Comment:
@ColinRobbins Could you update the description here?

Reply:
I've made this change...